master/internal/command/command.go

```diff
@@ -71,7 +72,7 @@ type command struct {
 	exitStatus     *string
 	addresses      []scheduler.Address
 
-	cluster     *actor.Ref
+	rp          *actor.Ref
```

Comment:
```suggestion
	rps          *actor.Ref
```
It would be good to distinguish this one from the singular `rp` on task actors.

(Maybe even `resourceProviders`, though I'm on the fence about that.)